images.CI/macos/anka/Service.Helpers.psm1

```diff
@@ -302,7 +305,13 @@ function Invoke-SSHPassCommand {
         "${env:SSHUSER}@${HostName}"
     )
     $sshPassOptions = $sshArg -join " "
-    $result = bash -c "$sshPassOptions \""$Command\"" 2>&1"
+    if ($PSVersionTable.PSVersion.Major -eq 5) {
```

Comment:
this script is always being run on Mac hosts i.e. they do not have powershell other than push-core which is always 7 and never 5, do we need the native powershell condition?

Reply:
Good point, thank you